ch2driver/pytpcc/drivers/nestcollectionsdriver.py

```diff
@@ -995,11 +1199,13 @@ def doNewOrder(self, params):
             s_order_cnt = stockInfo[0]["s_order_cnt"]
             s_remote_cnt = stockInfo[0]["s_remote_cnt"]
             s_data = stockInfo[0]["s_data"]
-            distxx = "s_dist_" + str(d_id).zfill(2)
+            #distxx = "s_dist_" + str(d_id).zfill(2)
             # print "NewOrder Stage #4.01"
             # print distxx
             # print stockInfo[0][distxx]
-            s_dist_xx = stockInfo[0][distxx] # Fetches data from the s_dist_[d_id] column
+            #s_dist_xx = stockInfo[0][distxx] # Fetches data from the s_dist_[d_id] column
```

Comment:
Can we remove these commented-out lines?

Reply:
Done